packaging/debian/control.base

```diff
@@ -6,4 +6,8 @@ Architecture: all
 Essential: no
 Maintainer: github.com/Shopify/shopify-app-cli
 Depends: ruby (>= 2.3)
-Description: Shopify CLI metapackage, which installs the shopify-cli Ruby gem
+Description:
+  Shopify CLI helps you build Shopify apps faster. It quickly scaffolds Node.js
+  and Ruby-on-Rails embedded apps. It also automates many common tasks in the
```

Comment:
Minor: I'd avoid the hyphenation here.

Reply:
Oh, I see this is existing content, so treat this as optional.

Reply:
Still worth fixing, IMHO.